src/name_bazaar/server/db_sync.cljs

```diff
@@ -53,45 +54,53 @@
-    (update :bid/datetime bn/->number)
-    (assoc :bid/offering offering)
-    (->> (db/insert-bid! (state/db server-state)))))
+(defn on-offering-bid [server-state err {{:keys [:offering :version :extra-data] :as args} :args}]
+  (logging/info ::on-offering-bid "Handling blockchain event" {:args args})
+  (try 
+    (-> (zipmap [:bid/bidder :bid/value :bid/datetime] extra-data)
+        (update :bid/bidder (comp prepend-address-zeros web3/from-decimal))
+        (update :bid/value bn/->number)
+        (update :bid/datetime bn/->number)
+        (assoc :bid/offering offering)
+        (->> (db/insert-bid! (state/db server-state))))
+    (catch :default e
+      (logging/error ::on-offering-bid "Error handling blockchain event" {:error e}))))
 
 (defn stop-watching-filters! []
   (doseq [filter @event-filters]
     (when filter
       (web3-eth/stop-watching! filter (fn [])))))
 
-(defn on-request-added [server-state err {{:keys [:node :round :requesters-count]} :args}]
+(defn on-request-added [server-state err {{:keys [:node :round :requesters-count] :as args} :args}]
+  (logging/info ::on-request-added "Handling block event" {:args args})
   (db/upsert-offering-requests-rounds! (state/db server-state)
                                        {:offering-request/node node
                                         :offering-request/round (bn/->number round)
                                         :offering-request/requesters-count (bn/->number requesters-count)}))
 
-(defn on-round-changed [server-state err {{:keys [:node :latest-round]} :args}]
-  (go
+(defn on-round-changed [server-state err {{:keys [:node :latest-round] :as args} :args}]
+  (logging/info ::on-round-changed "Handling block event" {:args args})
+  (gotry
     (let [latest-round (bn/->number latest-round)
           request (second (<! (offering-requests/get-request server-state {:offering-request/node node})))]
       (db/upsert-offering-requests! (state/db server-state)
                                     (-> request
-                                      (assoc :offering-request/latest-round latest-round)))
+                                        (assoc :offering-request/latest-round latest-round)))
       (when (= latest-round (:offering-request/latest-round request))
         ;; This is optimisation so we don't have to go through all on-request-added from block 0
         ;; We just save current count of latest round, because it's all we need. Don't need all history
         (on-request-added server-state nil {:args {:node node
                                                    :round latest-round
                                                    :requesters-count (:offering-request/requesters-count request)}})))))
 
-(defn on-ens-new-owner [server-state err {{:keys [:node :owner]} :args}]
-  (go
+(defn on-ens-new-owner [server-state err {{:keys [:node :owner] :as args} :args}]
+  (logging/info ::on-ens-new-owner "Handling blockchain event" {:args args})
```

Comment:
This one is enough to log only after `(when offering ...` below. Logging all ENS events would just bloat our logs